assets/js/modules/analytics-4/datastore/audiences.js

```diff
@@ -522,6 +523,16 @@ const baseActions = {
 			return { error };
 		}
 
+		// Expire new badges for initially configured audiences.
+		yield commonActions.await(
+			dispatch( CORE_USER ).setExpirableItemTimers(
+				configuredAudiences.map( ( slug ) => ( {
+					slug: `${ AUDIENCE_ITEM_NEW_BADGE_SLUG_PREFIX }${ slug }`,
+					expiresInSeconds: 1,
```

Comment:
The `expirable-items` infrastructure does not accept `0` as a value for `expiresInSeconds`, so we're using `1` so that it expires the new badges as soon as possible, i.e. 1 second.